frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -224,54 +207,53 @@ export default {
       if (this.sentencesOrigin === "Prediction") {
         return this.predictions;
       }
+      if (this.sentencesOrigin === "Preannotation") {
+        return this.predictions;
+      }
```

Comment:
1/replace the 3 if by a swith case
2/ to not have case problem in case this.sentencesOrigin change, compare the same case text